templates/todo/projects/java-mongo-aca/.repo/bicep/infra/main.bicep

```diff
@@ -40,9 +40,6 @@ param apimSku string = 'Consumption'
 @description('Id of the user or app to assign application roles')
 param principalId string = ''
 
-@description('The base URL used by the web service for sending API requests')
-param webApiBaseUrl string = ''
```

Comment:
The linter was complaining that this parameter was unused, which is true.  So I removed it from here and from the associated `main.parameters.json` file.